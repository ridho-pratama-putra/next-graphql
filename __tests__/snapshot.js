import { render } from '@testing-library/react'
import Index from '../pages'

it('renders homepage unchanged', () => {
    const { container } = render(<Index />)
    expect(container).toMatchSnapshot()
})