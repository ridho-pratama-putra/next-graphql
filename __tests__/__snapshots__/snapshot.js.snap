// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders homepage unchanged 1`] = `
<div>
  <div
    class="jsx-25781291bf750 container"
  >
    <main
      class="jsx-25781291bf750"
    >
      <h1
        class="jsx-25781291bf750 title"
      >
        Welcome to 
        <a
          class="jsx-25781291bf750"
          href="https://nextjs.org"
        >
          Next.js!
        </a>
      </h1>
      <p
        class="jsx-25781291bf750 description"
      >
        Get started by editing 
        <code
          class="jsx-25781291bf750"
        >
          pages/index.js
        </code>
      </p>
      <div
        class="jsx-25781291bf750 grid"
      >
        <a
          class="card"
          href="/posts/first-post"
        >
          <h3
            class="jsx-25781291bf750"
          >
            First Post →
          </h3>
          <p
            class="jsx-25781291bf750"
          >
            implementation of getStaticProps.
          </p>
        </a>
        <a
          class="card"
          href="/posts/second-post"
        >
          <h3
            class="jsx-25781291bf750"
          >
            Second Post →
          </h3>
          <p
            class="jsx-25781291bf750"
          >
            implementation of getServerSideProps.
          </p>
        </a>
        <a
          class="card"
          href="/posts/third-post"
        >
          <h3
            class="jsx-25781291bf750"
          >
            dynamic routes Post 3 →
          </h3>
          <p
            class="jsx-25781291bf750"
          >
            implementation of getStaticProps
          </p>
        </a>
        <a
          class="card"
          href="/posts/fourth-post"
        >
          <h3
            class="jsx-25781291bf750"
          >
            dynamic routes Post 4 →
          </h3>
          <p
            class="jsx-25781291bf750"
          >
            implementation of getStaticProps
          </p>
        </a>
        <a
          class="card"
          href="/posts/fifth-post"
        >
          <h3
            class="jsx-25781291bf750"
          >
            dynamic routes Post 5 →
          </h3>
          <p
            class="jsx-25781291bf750"
          >
            implementation of getStaticProps.
          </p>
        </a>
        <a
          class="card"
          href="/posts/sicth-post"
        >
          <h3
            class="jsx-25781291bf750"
          >
            dynamic routes Post sicth →
          </h3>
          <p
            class="jsx-25781291bf750"
          >
            implementation of getStaticProps.
          </p>
        </a>
        <a
          class="card"
          href="/forms/release-form"
        >
          <h3
            class="jsx-25781291bf750"
          >
            Release forms →
          </h3>
          <p
            class="jsx-25781291bf750"
          >
            implementation of submit chunked type media.
          </p>
        </a>
      </div>
    </main>
    <footer
      class="jsx-25781291bf750"
    >
      <a
        class="jsx-25781291bf750"
        href="https://vercel.com?utm_source=create-next-app&utm_medium=default-template&utm_campaign=create-next-app"
        rel="noopener noreferrer"
        target="_blank"
      >
        Powered by
         
        <img
          alt="Vercel"
          class="jsx-25781291bf750 logo"
          src="/vercel.svg"
        />
      </a>
    </footer>
  </div>
</div>
`;
